downstream/assemblies/aap-hardening/assembly-hardening-aap.adoc

```diff
@@ -1,16 +1,19 @@
 ifdef::context[:parent-context: {context}]
 
 [id="hardening-aap"]
-= Hardening {PlatformNameShort}
+== Hardening {PlatformNameShort}
 
 :context: hardening-aap
 
 [role="_abstract"]
+This guide takes a practical approach to hardening the {PlatformNameShort} security posture, starting with the planning and architecture phase of deployment and then covering specific guidance for the installation phase.  As this guide specifically covers {PlatformNameShort} running on Red Hat Enterprise Linux (RHEL), hardening guidance for RHEL will be covered where it affects the automation platform components.
 
+include::aap-hardening/con-hardening-guide-audience.adoc[leveloffset=+2]
+include::aap-hardening/con-product-overview.adoc[leveloffset=+2]
+include::aap-hardening/con-platform-components.adoc[leveloffset=+2]
```

Comment:
The leveloffset value for the first included file needs to be =+1

Reply:
Thank you, @ariordan-redhat! That's so weird because I originally placed one = character and the level offset value was =+1 for the include file and I was still getting the error. The changed the values while troubleshooting. I guess I should have left them alone. 

Thanks, agin for the second pair of 👀 on this!

Reply:
Yes, I guessed that they were experiments. I used Asciidoctor and bccutil to build the doc, BTW

Reply:
Okay, I was using asciidoctor at that time. I'll be using both from now on.

Reply:
For once, both Asciidoctor and bccutil had the same error.

Reply:
Changes made. Can you please review @ariordan-redhat?